From 864d718437b29eaf828f7ebfb47c5b8b76a98f5e Mon Sep 17 00:00:00 2001
From: pjgowtham <pjgowtham@gmail.com>
Date: Sat, 14 Oct 2023 14:19:31 +0530
Subject: [PATCH] spartan: Disable logging sensors-hal events

This sets log_level to 0
10-14 14:17:17.674 14629 14629 I sensors-hal: get_system_config:67, log_level: 0

This is done especially to disable spammy logging of oplus ambient light sensor events
10-14 14:17:12.821 12603 14590 I sensors-hal: handle_oplus_ambient_light_event:157, oplus_ambient_light: ts=3858719100254, lux=11, data_c=0, delta_ms=0, data_cnt=0

Change-Id: Ib097c172bb01ae62dec542a76b289a9641764bc2
Signed-off-by: mukesh22584 <mks22584@gmail.com>
Signed-off-by: DragonEmperor9480 <amruteshnaregal1234@gmail.com>
---
 vendor.prop | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/vendor.prop b/vendor.prop
index 65adcaa..6cbcad4 100644
--- a/vendor.prop
+++ b/vendor.prop
@@ -218,6 +218,8 @@ ro.telephony.default_network=33,22
 
 # Sensors
 persist.vendor.sensors.allow_non_default_discovery=true
+persist.vendor.sensors.debug.hal=0
+persist.vendor.sensors.on_change_sample_period=true
 persist.vendor.sensors.sync_request=true
 
 # USB
